config_ui.go

```diff
@@ -23,6 +23,7 @@ type UIConfig struct {
 	CORSOrigins             []string `bson:"cors_origins" json:"cors_origins" yaml:"cors_origins"`         // allowed request origins for some UI Routes
 	LoginDomain             string   `bson:"login_domain" json:"login_domain" yaml:"login_domain"`         // domain for the login cookie (defaults to domain of app)
 	ExpireLoginCookieDomain string   `bson:"expire_domain" json:"expire_domain" yaml:"expire_domain"`      // expire login token for this domain when the user logs in
+	UserVoice               string   `bson:"user_voice" json:"user_voice" yaml:"user_voice"`
```

Comment:
super nit but I think UserVoice is just 1 word, so I would remove the underscore in these tags as well as in the rest model